install/migrations/update_10.0.3_to_10.0.4/inventory.php

```diff
@@ -47,3 +47,6 @@
 
 //new right value for locked_field (previously based on config UPDATE)
 $migration->addRight('locked_field', CREATE | PURGE, ['config' => UPDATE]);
+
+//add date_install
+$migration->addField(getTableForItemType(Item_OperatingSystem::class), 'install_date', 'date');
```

Comment:
`getTableForItemType(Item_OperatingSystem::class)` should be replaced by `glpi_items_operatingsystems`